workflows/diagnostics/tests/prognostic/test_compute.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import xarray as xr
+
+from fv3net.diagnostics.prognostic_run.compute import itcz_edges
+
+
+def test_itcz_edges():
```

Comment:
Hmm maybe add a test case that doesn't just depend on the hardcoded edges of the ITCZ, i.e., has a min or max within it?

Reply:
Sure.